README.md

```diff
@@ -99,6 +99,7 @@ uphold this code.**
 tracking requests and bugs, please see
 [TensorFlow Discuss](https://groups.google.com/a/tensorflow.org/forum/#!forum/discuss)
 for general questions and discussion, and please direct specific questions to
+[TensorFlow Forum](https://discuss.tensorflow.org/) or
```

Comment:
Can we just remove the old reference to  https://groups.google.com/a/tensorflow.org/forum/#!forum/discuss ?
The forum was meant to replace that.

Reply:
@MarkDaoust ,Thank you for reviewing.

Yes. Just now updated as per your suggestion.